src/app/pages/charging-stations/registration-tokens/registration-token/charging-stations-registration-token.component.ts

```diff
@@ -128,7 +129,13 @@ export class ChargingStationsRegistrationTokenComponent implements OnInit {
       }
     }, (error) => {
       this.spinnerService.hide();
-      Utils.handleHttpError(error, this.router, this.messageService, this.centralServerService, 'tenants.create_error');
+      switch (error.status) {
+        case HTTPError.BAD_REQUEST:
```

Comment:
It's not BAD_REQUEST I think, it's FORBIDDEN ;-)

Reply:
modified to forbidden